crates/shared/src/sources/balancer_v2/pools/composable_stable.rs

```diff
@@ -100,94 +97,9 @@ mod tests {
     use {
         super::*,
         crate::sources::balancer_v2::graph_api::Token,
-        contracts::dummy_contract,
         ethcontract::{H160, H256},
-        ethcontract_mock::Mock,
-        futures::future,
-        maplit::btreemap,
     };
 
-    #[tokio::test]
```

Comment:
Why did you want to remove all these tests?

Reply:
These were one of the tests that tested implementation details but not really the outcome of a function call which make them less useful. Basically whenever you adjust the implementation you have to refactor lots of code which makes those tests very fragile.
It makes more sense to me to tests the logic using an e2e test that fetches liquidity that was deployed on a test new.